veggieseasons/lib/screens/details.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/widgets.dart';
+
+class DetailsScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        middle: Text('Details'),
+      ),
+      backgroundColor: Color(0xfff0f0f0),
```

Comment:
same on the hex value here. Maybe put all these in an "iOSColors.dart" file? or add to that styles.dart file.

Reply:
I've got a styles.dart class going that I hope to replace with something more sophisticated in a future release. I'll put it in there for now.